src/UIComponents/DataDisplay.jsx

```diff
@@ -4,18 +4,15 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import Statement from "./Statement";
 import DataTable from "./DataTable";
-import { setCurrentColumn } from "../redux";
 import { styles } from "../constants";
 
 class DataDisplay extends Component {
   static propTypes = {
-    data: PropTypes.array,
-    setCurrentColumn: PropTypes.func,
-    setColumnRef: PropTypes.func
+    data: PropTypes.array
   };
 
   render() {
-    const { data, setCurrentColumn, setColumnRef } = this.props;
```

Comment:
I like that this simplifies both the UI and the code by no longer having to deal with the refs